independent-publisher-connectors/OpenAI/apiDefinition.swagger.json

```diff
@@ -0,0 +1,213 @@
+{
+  "swagger": "2.0",
+  "info": {
+    "title": "OpenAI",
+    "description": "Connect to the OpenAI API and use the Power of GPT3, API key must be entered as \"Bearer YOUR_API_KEY\"",
+    "version": "1.0"
+  },
```

Comment:
Hello @PowerRobin,
I hope you are doing well.
Can you please add the Contact section? Please see below.
"contact": {
      "name": "Your Name",
      "url": "Your URL/ Connector/ api URL",
      "email": "Your email address"
    }
We have a weekly connector telemetry report for our independent publishers. To start getting this report, please add your email in the swagger file and submit a quick update.
Please let me know if you have any questions.
Thank you so much for you hard work with us.
Best regards,
Amjed Ayoub